setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
```

Comment:
It would be nice to have `dataclasses` in `install_requires` for `pip install -e .`

    @@ -26,6 +26,7 @@ setup(
         install_requires=[
             "colorlog",
             "astor",
    +        "dataclasses",
             "six",
             "mako",
             "pyverilog",

Reply:
Done